### ISSUE TITLE
Ruler settings interact badly

### ISSUE THREAD

**Issue by dgl** [mikewest/vimroom#3]
I have something like:

  set t_mr=^[[0;1;37;44m
  set ruler

in my .vimrc to show a coloured ruler, this doesn't make much sense with this. If I manually :set noruler t_mr= and do a ^L after entering this it works.

[I'm http://news.ycombinator.com/item?id=3088408]

**Comment by mikewest** [comment]
Ok. Let me just look up what `t_mr` does, and I'll see what I can do. :)

It should be possible to save it off to a temp variable and replace it after toggling Vimroom. Trying that out now. Thanks for the bug report!

**Comment by mikewest** [comment]
Can you pull the current `master` branch and let me know if the quick fix actually takes care of the bug?

Thanks!

**Comment by dgl** [comment]
Yes, that seems to work to hide the not active parts so it looks much better.

I realised I also have a :set highlight=[...,]sr[,...] type thing in my .vimrc which is probably what is making t_mr do that to start with, sorry for simplifying too much...

**Comment by mikewest** [comment]
So the patch fixes the bug for you? Or is there still something in your highlight string that Vimroom is having issues with?

**Comment by dgl** [comment]
It still shows the filename at the bottom, but it's cleaner.

I think this should reproduce what I see:
vim -u /dev/null --cmd 'set highlight=sr' -c VimroomToggle

I think ideally 'sr' should be filtered out of highlight, then the need for the t_mr saving I originally suggested should go away.
